cypress/support/index.js

```diff
@@ -0,0 +1,21 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import "@testing-library/cypress/add-commands";
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```

Comment:
This is where I added @testing-library which gives the ability to use @testing-library type syntax similar to how we write unit tests. https://github.com/testing-library/cypress-testing-library/tree/main/types/index.d.ts 
Aka DOM Testing Library